score-jClouds/src/main/java/io/cloudslang/content/jclouds/services/impl/imagesImpl/AmazonImageService.java

```diff
@@ -0,0 +1,64 @@
+package io.cloudslang.content.jclouds.services.impl.imagesImpl;
```

Comment:
why do you need imagesImpl package? why not keep them in the same io.cloudslang.content.jclouds.services.impl package?